charts/nfs-server-provisioner/templates/statefulset.yaml

```
@@ -117,6 +117,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.priorityClass.name }}
+      priorityClassName: {{ . }}
```

In the edge case you have a numerical string set, this will error as `{{ . }}` will emit the string without quotes, and then k8s will complain as when it parses the YAML, it will see it as a numerical value.

I can almost imagine someone having a PriorityClass named `5` or so, but yeah, this is sure is an edge case. I care more about the helping establish a common good practice where it could matter.

```suggestion
      priorityClassName: {{ . | quote }}
```